firmware/quadruna/CRIT/src/app/app_leds.c

```diff
@@ -27,8 +49,9 @@ void app_leds_update(void)
     const bool torquevec_light_on = app_canRx_VC_TorqueVectoringEnabled_get();
     io_led_torquevec_set(torquevec_light_on);
 
-    const BoardLEDStatus shutdown_sensor = OK; // TODO implement VC last shutdown node code is in
-    io_led_shutdown_set(shutdown_sensor);
+    // or driven by BMS_drive_state???
+    const BoardLEDStatus shutdown_sensor_ok = app_canRx_VC_FirstFaultNode_get() == SHDN_OK ? OK : FAULT;
+    io_led_shutdown_set(shutdown_sensor_ok);
```

Comment:
Technically out of scope of this PR but `BoardLEDStatus` entries should be prefixed with the enum type, i.e. `BOARD_LED_STATUS_OK`. Very easy for OK/FAULT to cause mysterious bugs due to name collisions down the line.